WordPress/Classes/ViewRelated/Blog/Blog List/BlogListViewController.m

```diff
@@ -1,1073 +0,0 @@
-        self.navigationItem.leftBarButtonItem.accessibilityIdentifier = @"my-sites-cancel-button";
-    }
-    
-
-    self.navigationItem.title = self.configuration.navigationTitle;
-}
-
-- (void)cancelTapped
-{
-    [self dismissViewControllerAnimated:YES completion:nil];
-}
-
-- (NSString *)modelIdentifierForElementAtIndexPath:(NSIndexPath *)indexPath inView:(UIView *)view
-{
-    if (!indexPath || !view) {
-        return nil;
-    }
-
-    // Preserve objectID
-    NSManagedObject *managedObject = [self.dataSource blogAtIndexPath:indexPath];
-    return [[managedObject.objectID URIRepresentation] absoluteString];
-}
-
-- (NSIndexPath *)indexPathForElementWithModelIdentifier:(NSString *)identifier inView:(UIView *)view
-{
-    if (!identifier || !view) {
-        return nil;
-    }
-
-    // Map objectID back to indexPath
-    NSManagedObjectContext *context = [[ContextManager sharedInstance] mainContext];
-    NSManagedObjectID *objectID = [context.persistentStoreCoordinator managedObjectIDForURIRepresentation:[NSURL URLWithString:identifier]];
-    if (!objectID) {
-        return nil;
-    }
-
-    NSError *error = nil;
-    Blog *blog = (Blog *)[context existingObjectWithID:objectID error:&error];
-    if (error || !blog) {
-        return nil;
-    }
-
-    NSIndexPath *indexPath = [self.dataSource indexPathForBlog:blog];
-
-    return indexPath;
-}
-
-- (void)viewDidLoad
-{
-    [super viewDidLoad];
-
-    [self configureStackView];
-
-    [self configureSearchBar];
-    [self.stackView addArrangedSubview:self.searchBar];
-
-    [self configureTableView];
-    [self.stackView addArrangedSubview:self.tableView];
-
-    self.editButtonItem.accessibilityIdentifier = NSLocalizedString(@"Edit", @"");
-
-    [self registerForAccountChangeNotification];
-    [self registerForPostSignUpNotifications];
-
-    [WPAnalytics trackEvent:WPAnalyticsEventSiteSwitcherDisplayed];
-}
-
-- (void)viewWillAppear:(BOOL)animated
-{
-    [super viewWillAppear:animated];
-    [self registerForKeyboardNotifications];
-
-    self.visible = YES;
-    [self.tableView reloadData];
-    [self updateBarButtons];
-    [self updateSearchVisibility];
-    [self maybeShowNUX];
-    [self updateViewsForCurrentSiteCount];
-    [self validateBlogDetailsViewController];
-    [self syncBlogs];
-    [self updateCurrentBlogSelection];
-}
-
-- (void)viewDidAppear:(BOOL)animated
-{
-    [super viewDidAppear:animated];
-    [self createUserActivity];
-}
-
-- (void)viewWillDisappear:(BOOL)animated
-{
-    [super viewWillDisappear:animated];
-    [self unregisterForKeyboardNotifications];
-    if (self.searchBar.isFirstResponder) {
-        [self.searchBar resignFirstResponder];
-    }
-    self.visible = NO;
-    
-    [SVProgressHUD resetOffsetFromCenter];
-    [SVProgressHUD setDefaultMaskType:SVProgressHUDMaskTypeNone];
-
-    [WPAnalytics trackEvent:WPAnalyticsEventSiteSwitcherDismissed];
```

Comment:
Does the new SwiftUI view continue to track the same events as the legacy `BlogListViewController`?

Reply:
I'm adding the events here https://github.com/wordpress-mobile/WordPress-iOS/pull/23455.